docs/models/basics.mdx

```diff
@@ -29,10 +29,10 @@ You can configure as many model providers as you like. These models will be show
 side-by-side comparison view in the web reporter.
 
 ```json empiricalrc.json
+"provider": "openai",
```

Comment:
hey, the intent here was to replace the `runs` array with something else (e.g. providers). it's a bit confusing to have `runs` in the config, and also "run" in the cli.

Reply:
My bad, misinterpreted the task title